split

```diff
@@ -0,0 +1,83 @@
+const { splitInteger } = require('./splitInteger'); // Replace './splitInteger' with your actual import path
+
+describe('splitInteger', () => {
+  // Basic Cases
+  it('should split 8 into 1 part', () => {
```

Comment:
There were some good test descriptions - you should have followed those examples. The test name is basically like a test case. Here is a good example:
"should split a number into equal parts if a value is divisible by a numberOfParts"